tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/cluster/distribution/NettySymmetricClusterTest.java

```diff
@@ -16,10 +16,79 @@
  */
 package org.apache.activemq.artemis.tests.integration.cluster.distribution;
 
+import org.apache.activemq.artemis.api.core.client.ActiveMQClient;
+import org.apache.activemq.artemis.api.core.client.ClientSessionFactory;
+import org.apache.activemq.artemis.api.core.client.ServerLocator;
+import org.junit.Test;
+
 public class NettySymmetricClusterTest extends SymmetricClusterTest {
 
    @Override
    protected boolean isNetty() {
       return true;
    }
+
+   @Test
+   public void testConnectionLoadBalancingUsingInitialConnectors() throws Exception {
+      final String ADDRESS = "queues.testaddress";
+      final String QUEUE = "queue0";
+      final String URL = "(tcp://localhost:61616,tcp://localhost:61617)";
```

Comment:
@jbertram I'm expecting it to be easier to set this through the URI as a property.. it would be nice to make sure it works.